main.tf

```diff
@@ -21,7 +17,6 @@ resource "aws_vpn_gateway" "this" {
 }
 
 resource "aws_customer_gateway" "this" {
-  count = var.create_vpn_connection ? length(var.cgw_ip_addresses) : 0
```

Comment:
We're only removing `var.create_vpn_connection`. We still need a customer gateway per CGW IP.

Reply:
Gotcha. So, the conditional is being removed entirely? 

`count = length(var.cgw_ip_addresses)`

Reply:
That's correct, yes. I should have left it as a suggestion, sorry...